sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3807,11 +3798,6 @@ pi_result piSamplerCreate(pi_context Context,
         case PI_SAMPLER_ADDRESSING_MODE_MIRRORED_REPEAT:
           ZeSamplerDesc.addressMode = ZE_SAMPLER_ADDRESS_MODE_MIRROR;
           break;
-        default:
```

Comment:
why didn't you handle this like in other places?

Reply:
Sergey's original simply removed it.  The other clauses in this particular switch aren't exiting they are setting an addressMode.  So, I thought it safe to return to what he had done originally, rather than insert a new untested premature exit.  

But I can do that, if you'd prefer. I'm just not familiar with this expanse of code and am trying not to be disruptive.

Reply:
I think removing "default" is dangerous in general, so I'd prefer to allow it (by disabling the warning in the build), or move it after the switch.

Reply:
I may have misspoke. We can't simply return PI_INVALID_VALUE after the switch, because the other cases all use `break` so all cases will end up after the switch, they don't exit.     So, if the default must be removed, there aren't a lot of good options.  They are all ugly.